src/main/webapp/deletefile.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+    <link rel="stylesheet" href="deletefile.css">
+    <!-- Bootsrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" 
+    integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <script src="script.js"></script>
```

Comment:
Please rename "script.js" to something more meaningful. I don't see that script anywhere, so maybe this is just accidental copypasta?

Reply:
+1, please rename to deletescript.js (or delete-script.js if you are introducing hyphenated file names).